Makefile

```diff
@@ -284,7 +284,7 @@ ifneq ($(_WITH_CLOUSEAU), )
 		"$(_WITH_CLOUSEAU)" \
 		"$(TEST_OPTS)" \
 		--locald-config test/config/test-config.ini \
-		--no-eval 'mix test --trace --include test/elixir/test/config/search.elixir'
+		--no-eval 'mix test --trace --include test/elixir/test/config/search.elixir $(EXUNIT_OPTS)'
```

Comment:
Do we want to commit this?
If yes, is this a separate commit?

Reply:
This was before we added it to the initial branch. I removed the commit now, after the rebase